reflectable/lib/src/builder_implementation.dart

```diff
@@ -2774,8 +2846,8 @@ class _ClassDomain {
 
   /// Finds all instance members by going through the class hierarchy.
   Iterable<ExecutableElement> get _instanceMembers {
-    Map<String, ExecutableElement> helper(ClassElement classElement) {
-      var member = _reflectorDomain._instanceMemberCache[classElement];
+    Map<String, ExecutableElement> helper(InterfaceElement interfaceElement) {
+      var member = _reflectorDomain._instanceMemberCache[interfaceElement];
```

Comment:
Maybe this could be written more clearly with
```suggestion
      return _reflectorDomain._instanceMemberCache.putIfAbsent(interfaceElement, () {
      ....
      
   }
```